sycl/test-e2e/Matrix/SG32/joint_matrix_int8_colmajorA_colmajorB.cpp

```diff
@@ -18,7 +18,7 @@
 // This tests support of col major layout for matrix B which does transpose and
 // then VNNI transform. This is currently only available on AMX
 
-// XFAIL: gpu
+// XFAIL: gpu-intel-dg2 || arch-intel_gpu_bmg_g21
 // XFAIL-TRACKER: GSD-5768
```

Comment:
The BMG part is fine.
But for dg2, we need add:
"
// SG size = 32 is not currently supported for SYCL Joint Matrix by IGC on DG2
// UNSUPPORTED: gpu-intel-dg2
"

Reply:
just pushed a new commit that should address this, thanks!